src/main/cheat.c

```diff
@@ -466,7 +469,7 @@ static int cheat_parse_hacks_code(char *code, m64p_cheat_code **hack)
     while ((token = strtok_compat(input, ",", &saveptr))) {
         input = NULL;
 
-        ret = sscanf(token, "%08X %04X", &hackbuf[num_codes].address,
+        ret = sscanf(token, "%08" SCNx32 " %04X", &hackbuf[num_codes].address,
```

Comment:
Yeah, a `scanf` format string considers lower-case and upper-case hex input to fit both the `"%x"` and the `"%X"` format. In addition, there's not even a `SCNX32` macro defined in `<inttypes.h>` in the standard, only `SCNx32`.